docs/glossary.tsv

```diff
@@ -0,0 +1,46 @@
+Data Model / Schema	A data model is a formal representation of a domain which identified key entities and relations between them.	ISO11179 data model	Phillipe		
+Data Variable	A data variable is an information entity which can be defined in a data dictionary, which has a type and which can assume a range of values.	handedness	Phillipe		
+Dataset	A dataset in the context of the FAIR-DSM model is a data structure that is purposed for FAIR sharing. A dataset in FAIR-DSM-L-1 is a flexible container for any collection of data that is expected to be hosted and served for FAIR data sharing. Starting with FAIR-DSM-L-2 to FAIR-DSM-L-5, a dataset is expected to refer to a container for structured data, composed of a set of fields and each field is composed of a set of values (value set). In the majority of cases a dataset will be a tabular data structure (delimited text file) or a hierarchical data structure (json or xml)				
+Dataset Descriptor	A data descriptor is a data structure that contains any information that describes the Dataset, i.e., it is the datas structure representing the dataset's metadata		Phillipe		Metadata
+Dataset Field	A dataset field is an attribute of the dataset, which may be defined in the dataset model	creation_date	Phillipe		dataset attribute
+Dataset Field Values	Dataset field values is a collection of information entities which are allowed for a given dataset field.	{male,female}	Phillipe		value sets
+Dataset Model	A dataset model is a formal representation devised for a dataset	HCA,JSON	Phillipe		
+Domain Entity	A domain entity is a information element (a type, a concept) core to the description of a given knowledge area	MAGE-OM	Phillipe		
+Domain model	The domain model is a  formal representation of a knowledge domain with concepts, roles, datatypes, individuals, and rules, typically grounded in a description logic.		Yojana	WikiPedia (https://en.wikipedia.org/wiki/Domain_model) 	
+FAIR sharing	FAIR sharing is a data use case that is intended for a data object (dataset) to exhibit a state of FAIRness purposed for sharing and re-use				
+Field level metadata	The filed level metadata descibed in detail about the data stored in the field either based on the type of data, the usage of data, the data source etc.		Yojana		
+General purpose metadata schema	Generalised schemas require limited time and expertise, but the description may be inadequate. They include what are considered the essential elements to describe any data. Such schemas usually not domain-specific.	JSON,TSV,CSV	Yojana	Research data management (https://libguides.anu.edu.au/c.php?g=881167&p=6358464)	
+Generic Data Model	Generic data models are generalizations of conventional data models that define standardised general relation types, together with the data that may be related or connected by such a relation type.		Yojana	WikiPedia (https://en.wikipedia.org/wiki/Generic_data_model)	
```

Comment:
not sure this ads very much to the glossary

Reply:
Yes. But it is just added for completeness. :)